examples/browser-to-server/index.js

```diff
@@ -0,0 +1,39 @@
+import { createLibp2p } from 'libp2p'
+import { Noise } from '@chainsafe/libp2p-noise'
+import { multiaddr } from '@multiformats/multiaddr'
+import first from "it-first";
+import { pipe } from "it-pipe";
+import { fromString, toString } from "uint8arrays";
+import { webRTC } from 'js-libp2p-webrtc'
+
+  let stream;
+  const output = document.getElementById('output')
+  const sendSection = document.getElementById('send-section')
+  const appendOutput = (line) => output.innerText += `${line}\n`
+  const clean = (line) => line.replaceAll('\n', '')
+
+  const node = await createLibp2p({
+    transports: [webRTC()],
+    connectionEncryption: [() => new Noise()],
+  });
+  
+  await node.start()
```

Comment:
Does this even run? Since when can you have `await` in a non-async body, which the global body is AFAIK?